mirage/factories/registration-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { Factory, faker } from 'ember-cli-mirage';
+
+import RegistrationSchema from 'ember-osf-web/models/registration-schema';
+
+// eslint-disable-next-line no-spaced-func, func-call-spacing
+export default Factory.extend<RegistrationSchema & { schemaNoConflict: () => RegistrationSchema['schema'] }>({
```

Comment:
Might be cleaner to move `schemaNoConflict` to an interface, like you did with [traits in other factories](https://github.com/CenterForOpenScience/ember-osf-web/pull/383/files?utf8=%E2%9C%93&diff=unified&w=1#diff-8665aa44e6b9e35aa0df41fb6238db63R7).